docs_user/modules/proc_adopting-the-orchestration-service.adoc

```diff
@@ -2,64 +2,57 @@
 
 = Adopting the {orchestration}
 
-Adopting the {orchestration_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where {orchestration}
-is supposed to be disabled, should be patched to start the service with the
-configuration parameters provided by the source environment.
+To adopt the {orchestration_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR), where the {orchestration}
+is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
 
-After the adoption process has been completed, a user can expect that they
-will then have CRs for `Heat`, `HeatAPI`, `HeatEngine` and `HeatCFNAPI`.
-Additionally, a user should have endpoints created within {identity_service_first_ref} to facilitate
-the above mentioned servies.
-
-This guide also assumes that:
-
-. A {OpenStackPreviousInstaller} environment (the source Cloud) is running on one side;
-. A {rhocp_long} environment is running on the other side.
+After you complete the adoption process, you have CRs for `Heat`, `HeatAPI`, `HeatEngine`, and `HeatCFNAPI`, and endpoints within the {identity_service_first_ref} to facilitate these services.
```

Comment:
What do we mean by the endpoints “facilitate” the services? Connect to the services?

Reply:
Maybe: "After you complete the adoption process, you have CRs and endpoints within the {identity_service_first_ref} for `Heat`, `HeatAPI`, `HeatEngine`, and `HeatCFNAPI`." ?

Reply:
@bshephar What do we mean by the endpoints “facilitate” the services? Connect to the services?

Reply:
I mean the endpoints in Keystone:
`openstack endpoint list`

```
[m3@osp-df-3 ~]$ oc exec -it openstackclient -- openstack endpoint list --service orchestration -f yaml
- Enabled: true
  ID: 41bb5aa54d7446c2b0ec35ed957e74aa
  Interface: internal
  Region: regionOne
  Service Name: heat
  Service Type: orchestration
  URL: https://heat-api-internal.openstack.svc:8004/v1/%(tenant_id)s
- Enabled: true
  ID: 99a8179acb9e4247a1d24e7e754e42b4
  Interface: public
  Region: regionOne
  Service Name: heat
  Service Type: orchestration
  URL: https://heat-api-public-openstack.apps.ostest.test.metalkube.org/v1/%(tenant_id)s
```